Controllers/LoginController.php

```diff
@@ -32,6 +39,7 @@ public function __construct()
     {
         $this->userModel = new UserModel;
```

Comment:
A checker si c'est toujours valable https://www.php-fig.org/psr/psr-12/#4-classes-properties-and-methods